site/en/workflows.md

```diff
@@ -1399,6 +1535,11 @@ If your computer is not that powerful you may try this workflow. It works fine f
       <td>ocrd-tesserocr-segment-line</td>
```

Comment:
This second (now third) workflow recommendation should really be changed now. With the all-in-one segmentation for Tesseract, we don't need repair, region-level deskewing and clipping anymore. Just do `ocrd-tesserocr-segment` and then dewarping. Also, when you advertise this for slower hardware, skip the overly slow, redundant skimage-binarize.